content-src/components/ASRouterAdmin/ASRouterAdmin.jsx

```diff
@@ -25,10 +25,20 @@ function relativeTime(timestamp) {
   return new Date(timestamp).toLocaleString();
 }
 
+const OPT_OUT_PREF = "discoverystream.optOut.0";
+const LAYOUT_VARIANTS = {
+  "basic": "Basic default layout (on by default in nightly)",
+  "dev-test-all": "A little bit of everything. Good layout for testing all components",
```

Comment:
Not sure if we should add some more here, as it would need to get cleaned up if we don't want something anymore. `dev-test-feeds` might be a good candidate for now. The other ones like `survey-a` through `survey-d` seem less likely to include.

Reply:
Sure, I can add dev-test-feeds